src/borg/cache.py

```diff
@@ -277,7 +277,8 @@ def load(self):
         self.timestamp = self._config.get('cache', 'timestamp', fallback=None)
         self.key_type = self._config.get('cache', 'key_type', fallback=None)
         self.ignored_features = set(parse_stringified_list(self._config.get('cache', 'ignored_features', fallback='')))
-        self.mandatory_features = set(parse_stringified_list(self._config.get('cache', 'mandatory_features', fallback='')))
+        self.mandatory_features = set(parse_stringified_list(self._config.get('cache', 'mandatory_features',
+            fallback='')))
```

Comment:
more than 120 would also avoid such formatting, which is worse IMHO.

Reply:
In situations like this with 3+ levels of nested function calls, would it be better to use a temp variable, such as this:
`
  \# use a temp, easy to understand and comment.
  cfg = self._config.get('cache', 'mandatory_features', fallback='')   # get configuration
  self.mandatory_features = set(parse_stringified_list(cfg))   # parse config string and set as mandatory feature...

  \# vs 3 level deep nested function calls can be a pain to grok
  self.mandatory_features = set(parse_stringified_list(self._config.get('cache', 'mandatory_features',
                                      fallback='')))
`